verifier/server/app.js

```diff
@@ -50,15 +53,22 @@ const requestMap = new Map();
 // GetQR returns auth request
 async function getQR(req, res) {
   // Audience is verifier id
-  const sessionId = 1;
+  const sessionId = parseInt(req.query.sessionId || '1');
```

Comment:
Not sure we should keep this, but it's convenient to be able to respond to the different QRs separately. It's not really a client responsibility though -- this is just a quick hack. Maybe each example should have its own session and/or challenge IDs.